net-misc/oidentd/oidentd-3.0.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+DESCRIPTION="Another (RFC1413 compliant) ident daemon"
+HOMEPAGE="https://oidentd.janikrabe.com/"
+SRC_URI="https://files.janikrabe.com/pub/${PN}/releases/${PV}/${P}.tar.xz"
+
+LICENSE="BSD-2 GPL-2 LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="debug ipv6 masquerade selinux"
+
+DEPEND="masquerade? ( net-libs/libnetfilter_conntrack )"
+
+RDEPEND="
+	acct-user/oidentd
+	acct-group/oidentd
+	selinux? ( sec-policy/selinux-oident )
+	${DEPEND}"
+
```

Comment:
This requires sys-devel/flex and sys-devel/bison as BDEPEND.

Reply:
Added them. Good catch, thank you.